update_rate: 20
motion_model:
  vel_bias: 0
  vel_std: 0.0001
  delta_bias: 0
  delta_std: 0.000001
  x_bias: 0
  x_std: 0.0000001
  x_vel_std: 0.001
  y_bias: 0.0
  y_std: 0.000001
  y_vel_std: 0.001
  theta_bias: 0
  theta_std: 0.000001
fake_urg:
  update_rate: 10
  theta_discretization: 656
  z_short: 0.03
  z_max: 0.16
  z_blackout_max: 50
  z_rand: 0.01
  z_hit: 0.8
  z_sigma: 0.03
  tf_prefix: ""